packages/agreement-process/src/model/domain/models.ts

```diff
@@ -50,9 +48,3 @@ export const CompactEService = z.object({
   name: z.string(),
 });
 export type CompactEService = z.infer<typeof CompactEService>;
-
-export const CompactTenant = z.object({
```

Comment:
If I'm not wrong, this is a model used in API, it should remain here.
We can create a new model for the lifecycle package and leverage on duck typing?

Reply:
@galalest this [BFF PR](https://github.com/pagopa/interop-be-monorepo/pull/665/files#diff-68553658db5b69e8a152ae19368ee33420c1cf67794d1515fc4ae64bc56c8a58) introduces the new types, I have chosen to introduce these changes in the first service, besides Agreement, that uses these methods.

Reply:
sounds good, but `CompactTenant` should still remain here, right?

Reply:
To make it shareable with new package it was moved to `pagopa-interop-models` in `tenant.ts` file with related definitions.